src/presentation/http/schema/Note.ts

```diff
@@ -28,5 +29,8 @@ export const NoteSchema = {
         },
       },
     },
+    tools: {
+      type: 'array',
```

Comment:
can we specify array schema as well?

Reply:
each array element would be object with different properties as we have different keys (toolNames), so i think that would not help with validation. if we just specify object, that would create an error
![Uploading image.png…]()

Reply:
how about passing it via array of `{id, name}` object instead? 

```
tools: [{id, name}, {id, name}, ...]
```

Reply:
now we pass it as EditorTool[]